Formula/lib/libgrape-lite.rb

```diff
@@ -4,6 +4,7 @@ class LibgrapeLite < Formula
   url "https://github.com/alibaba/libgrape-lite/archive/refs/tags/v0.3.4.tar.gz"
   sha256 "4d7c08560d56fde4a407566fea5ea348cf8ea7df5dbcc3285dcbfe6d9e5d6ff7"
   license "Apache-2.0"
+  revision 1
```

Comment:
Wait, why does this need a `revision` bump?

Reply:
Just a desperate attempt to fix vineyard since I’m getting impatient with the CI cycle times. I’ll take it out in the next run (unless you want to merge and followup).

I think cppinsights on Linux is very close to working. I saw some of the patch in the downloaded source code, so I took it out, but I think only some of the patch was integrated in the release.

Reply:
Yes, please drop it.

`vineyard` just needs the same fix for `-lomp` on Linux in the `test` block:

https://github.com/lukeshingles/homebrew-core/blob/5b687cb48db9b1fb07f5bc8b3908d821b1bf0e26/Formula/l/llvm.rb#L673-L677

You should be able to do something like
```ruby
if OS.linux?
  ENV.append "LDFLAGS", "-L#{lib}/#{Hardware::CPU.arch}-unknown-linux-gnu"
  ENV.append "LDFLAGS", "-Wl,-rpath,#{lib}/#{Hardware::CPU.arch}-unknown-linux-gnu"
end
```

`apache-arrow` probably needs (assuming `llvm` is appropriately defined)
```ruby
ENV.append "LDFLAGS", "-Wl,-rpath,#{llvm.opt_lib}" if OS.linux?
```

I'll leave a comment for `pocl` shortly.

When you push again, please cherry-pick the commits from these PRs:

- https://github.com/Homebrew/homebrew-core/pull/191456
- https://github.com/Homebrew/homebrew-core/pull/165986
- https://github.com/Homebrew/homebrew-core/pull/191372

You should be able to just do
```
git fetch origin <commit-hash>
git cherry-pick <commit-hash>
```
from your PR branch.